aws_lambda_builders/workflows/ruby_bundler/bundler.py

```diff
@@ -66,6 +66,12 @@ def run(self, args, cwd=None):
                     self.osutils.remove_directory(check_dir)
             else:
                 # Bundler has relevant information in stdout, not stderr.
-                raise BundlerExecutionError(message=out.decode("utf8").strip())
+                if out and err:
+                    message_out = out.strip() + b"\n" + err
```

Comment:
nit: `os.linesep` could be used instead here in case Windows does something odd with newline formatting. Additionally we could also use fstrings for formatting